pkgs/development/python-modules/schemainspect/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+
+  sqlalchemy,
+  setuptools,
+
+  # pytestCheckHook,
+  # pytest-cov,
+  # pytest-xdist,
+  # pytest-sugar,
+}:
+buildPythonPackage rec {
+  pname = "schemainspect";
+  version = "3.1.1663587362";
+  disable = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-opWtVvehnAnl4e+fFtrb9jkuJhlstfBbWv5hPJnOdGg=";
+  };
+
+  nativeCheckInputs = [
+    # pytestCheckHook
+    # pytest-cov
```

Comment:
In general we don't test codestyle or coverage in nixpkgs. We care for correctness in behaviour and logic, not development metrics.

```suggestion
```